Dan13436/doc updates (#1)
dan13436/arcgis-maps-sdk-unity-samples: README.md

# Esri CloudXR Demo

![image](arcgis-maps-sdk-unity-samples.png)

> Here is a VR demo developed by ESRI within the Unity Engine. It is built off the [ArcGIS SDK Unity VRSample](https://github.com/Esri/arcgis-maps-sdk-unity-samples/tree/main/samples_project/Assets/SampleViewer/Samples/VRSample), incorporating more intractability and diverse features in order to utilize the demo in tandem with [Nvidia’s CloudXR Application](https://developer.nvidia.com/cloudxr-sdk). The main branch is configured to work with our ArcGIS plugin version 1.3.0. If you would like to build the application to a VR headset you would need to switch to the URPBuild branch since Androids do not support the HD Rendering Pipeline we have implemented in the main branch.

Reviewer: ```suggestion
Here is a VR demo developed by ESRI within the Unity Engine. It is built off the [ArcGIS SDK Unity VRSample](https://github.com/Esri/arcgis-maps-sdk-unity-samples/tree/main/samples_project/Assets/SampleViewer/Samples/VRSample), incorporating more interaction and diverse features in order to utilize the demo in tandem with [Nvidia’s CloudXR Application](https://developer.nvidia.com/cloudxr-sdk). The main branch is configured to work with our ArcGIS plugin version 1.3.0. If you would like to build the application to a VR headset you would need to switch to the URPBuild branch since Android does not support the HD Rendering Pipeline we use in the main branch.
```

![image](import-samples.png)

6. If there are still errors, locate the ArcGISMapsSDK Assembly Definition within the folder `Assets -> Samples -> ArcGIS Maps SDK for Unity -> 1.3.0 -> Sample Content -> Arc GIS Maps SDK.Samples (asmdef)` and add the `Unity.InputSystem` asmdef to the references. Make sure to click *Apply* at the bottom.

7. Launch Unity and open the `VR Sample` level.

> 8. In the heirarchy select the `ArcGISMap` Game Object and then in the inspector set your API Key. You can learn more about [API keys](https://developers.arcgis.com/documentation/mapping-apis-and-services/security/api-keys/) and [Accounts](https://developers.arcgis.com/documentation/mapping-apis-and-services/deployment/accounts/) in the _Mapping APIs and location services_ guide.

Reviewer: ```suggestion
8. In the hierarchy select the `ArcGISMap` Game Object. In the inspector set the API key on the ArcGISMap Component. You can learn more about [API keys](https://developers.arcgis.com/documentation/mapping-apis-and-services/security/api-keys/) and [Accounts](https://developers.arcgis.com/documentation/mapping-apis-and-services/deployment/accounts/) in the _Mapping APIs and location services_ guide.
```
